ses

```diff
@@ -201,10 +201,24 @@ section_header "Ceph log files"
 
 if [ -d /var/log/ceph ]; then
     mkdir -p $LOG/ceph/log/ceph
-    cp /var/log/ceph/* $LOG/ceph/log/ceph/
+    # Needs to be recursive, as if there's any containerized logs in
+    # files, they're under /var/log/ceph/$FSID/
+    cp -r /var/log/ceph/* $LOG/ceph/log/ceph/
     plugin_message "Ceph logs copied to ceph/log subdirectory"
 fi
 
+if [ -x /usr/sbin/cephadm ]; then
+    cephadm ls | jq -r '.[] | "\(.fsid) \(.name)"' |
+    while read fsid name ; do
+        mkdir -p $LOG/ceph/log/ceph/$fsid
+        # Individual daemon logs from journalctl named without the leading "ceph-"
+        # string to avoid any conflict with possible log files already copied.
+        plugin_command cephadm logs --fsid $fsid --name $name > $LOG/ceph/log/ceph/$fsid/$name.log 2>&1
```

Comment:
Can we avoid `$fsid` in the log path name? It just makes browsing the archive and processing with scripts harder.

I suppose the supportconfig operates in the assumption that the node is used for only one cluster. Otherwise we would need to tweak much more things to make the supportconfig support multicluster:  all commands like `ceph status` or `ceph daemon` should be per cluster then logging its output in the corresponding `$fsid` directory.

Reply:
I put `$fsid` in the log path there, because supportconfig is running on the VM host, and when in runs `cp -r /var/log/ceph/*` a few lines above, to collect any files that may be present, this already includes the FSID. So I kept it consistent with that.

You're right though, everything else assumes there's only one cluster, so having the FSID here in the log path is a bit... strange... from that perspective.

I guess the thing to do is get rid of FSID here, and also remove it when running `cp -r`?

The other alternative is of course to make supportconfig multi-cluster aware and have *everything* in an FSID path. Is there any sense in doing that work? (I'm leaning towards "no", because we don't expect people to run multiple clusters on the same hardware, but would welcome other opinions)

Reply:
Not sure what the roadmap for multiple clusters is, maybe putting everything under an fsid folder is acceptable? So we'd get supportconfig with a dir tree like `ceph/<fsid>/...`?

Reply:
I think making it multi-cluster aware is a bit of work and out of scope here. For this PR I would just remove fsid from all log paths. If we eventually decide the multi-cluster support, it will need tweaking anyway.

Reply:
Agreed. 0e675c7 drops the FSID from the log directories.